run-integration-tests.sh

```diff
@@ -0,0 +1,63 @@
+#/bin/bash
+#
+# ITs need the environment variable 'GITHUB_TOKEN' defined
+#
+
+if [ -z "$1" ]; then
+  echo "syntax: $0 [DEV|LTS|LATEST_RELEASE]"
+  exit 1
+fi
+
+set -ex
+SQ_VERSION=$1
+
+cd its
```

Comment:
Instead of this, probably you could add `-f its/pom.xml` to the `mvn` command. If this is needed, then move down in the script, right before it's actually needed. (It would be good to get rid of it, because `cd` inside scripts is not recommended in general.)